cypress/e2e/editor-actions/configure-actions.spec.cy.ts

```diff
@@ -13,11 +13,11 @@ describe('Configure editor actions', () => {
 
     it('Should see all custom actions by default in particular order', () => {
         YasqeSteps.getActionButtons().should('have.length', 5);
-        YasqeSteps.getActionButton(0).should('have.attr', 'title', 'Create saved query');
-        YasqeSteps.getActionButton(1).should('have.attr', 'title', 'Show saved queries');
-        YasqeSteps.getActionButton(2).should('have.attr', 'title', 'Get URL to current query');
-        YasqeSteps.getActionButton(3).should('have.attr', 'title', 'Include inferred data in results: ON');
-        YasqeSteps.getActionButton(4).should('have.attr', 'title', 'Expand results over owl:sameAs: ON');
+        YasqeSteps.getActionButton(0).parent().should('have.attr', 'data-tooltip', 'Create saved query');
```

Comment:
Create a new step methid for getting the tooltip and don't put such traversal methods in the tests